core/src/test/scala/unit/kafka/message/MessageTest.scala

```diff
@@ -57,7 +56,7 @@ class MessageTest extends JUnitSuite {
   }
 
   @Test
-  def testFieldValues {
+  def testFieldValues = {
```

Comment:
It was part of old patch, see related issue, so I did not want to discard the work of someone else.
Anyway, the procedural programming style is going away in later releases of scala (dotty), so there will not be possible to write:
`def foo(n: Int) { n+1 }`
The reason is (I think) that foo always returns `Unit`. Someone not familiar with scala would expect foo to return Int. I agree that adding explicit return type, i.e. `:Unit` would be much clearer.